frontend/src/main/scala/bloop/bsp/BloopBspServices.scala

```diff
@@ -454,44 +453,64 @@ final class BloopBspServices(
   }
 
   def test(params: bsp.TestParams): BspEndpointResponse[bsp.TestResult] = {
-    def test(
-        id: BuildTargetIdentifier,
-        project: Project,
-        state: State
-    ): Task[State] = {
-      val testFilter = TestInternals.parseFilters(Nil) // Don't support test only for now
-      val handler = new BspLoggingEventHandler(id, state.logger, client)
-      Tasks.test(state, List(project), Nil, testFilter, handler, false)
+    def parseParameters: Try[bsp.ScalaTestParams] = {
```

Comment:
`parseParameters` would indicate a side effect function. Maybe `parsedParameters`?

Reply:
I'd be fine with either `parseTestParams` or `readTestParams` 👍 I think using `parsedParameters` is misleading because it's a `def` while I would expect it to be a value.